index.html

```diff
@@ -5,15 +5,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <script src="https://kit.fontawesome.com/c5012398d3.js" crossorigin="anonymous"></script>
+    
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" type="text/css" media="screen and (min-device-width: 800px)" href="css/over800.css" />
 </head>
 <body>
+    <nav>
+        <img class="nav-img" src="img\karma-logo.svg"   alt="Karma Logo">
+        <label for="toggle">&#9776;</label>
+        <input type="checkbox" id="toggle"/>
+        <div class="nav-a-container">
+            <a class="nav-a-first"><strong>Karma</strong></a>
+            <a class="nav-a">How it Works</a>
+            <a class="nav-a">Store</a>
+            <a class="nav-a">Blog</a>
+            <a class="nav-a">Help</a>
+            <a class="nav-a">Login</a>
+        </div>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    </nav>
+
+    <article>
+        <header> 
+            <div class="header-div">
+                <div class="header-image">  </div>
```

Comment:
I think (though haven't tried) you should be able to get the background image showing by including it in the CSS of "header-div" or "header" - this would be better than it having its own element